humble/distribution.yaml

```diff
@@ -3746,6 +3746,12 @@ repositories:
       url: https://github.com/ros4hri/human_description.git
       version: humble-devel
     status: developed
+  husarion_components_description:
+    source:
+      type: git
+      url: https://github.com/husarion/husarion_components_description
```

Comment:
CI is failing, see https://github.com/ros/rosdistro/actions/runs/16218265231/job/45792546958?pr=46856. I think you need to add `.git` at the end of the URL here and in the other file